tests/product/base_product_case.py

```diff
@@ -44,25 +45,25 @@
 class BaseProductTestCase(BaseTestCase):
     STANDALONE_BARE_CLUSTER = 'bare'
     BARE_CLUSTER = 'bare'
-    PA_ONLY_CLUSTER = 'pa_only_standalone'
+    STANDALONE_PRESTO_ADMIN_CLUSTER = 'pa_only_standalone'
     STANDALONE_PRESTO_CLUSTER = 'presto'
 
-    PA_ONLY_YS_CLUSTER = 'pa_only_ys'
-    PA_SLIDER_CLUSTER = 'pa_slider'
+    YARN_SLIDER_PRESTO_ADMIN_CLUSTER = 'pa_only_ys'
+    YARN_SLIDER_PRESTO_ADMIN_AND_SLIDER_CLUSTER = 'pa_slider'
 
-    _cluster_types = {
+    cluster_types = {
         BARE_CLUSTER: [],
-        PA_ONLY_CLUSTER: [PrestoadminInstaller,
-                          StandaloneModeInstaller],
+        STANDALONE_PRESTO_ADMIN_CLUSTER: [PrestoadminInstaller,
+                                          StandaloneModeInstaller],
         STANDALONE_PRESTO_CLUSTER: [PrestoadminInstaller,
                                     StandaloneModeInstaller,
                                     TopologyInstaller,
                                     StandalonePrestoInstaller],
-        PA_ONLY_YS_CLUSTER: [PrestoadminInstaller,
-                             YarnSliderModeInstaller],
-        PA_SLIDER_CLUSTER: [PrestoadminInstaller,
-                            YarnSliderModeInstaller,
-                            SliderInstaller]
+        YARN_SLIDER_PRESTO_ADMIN_CLUSTER: [PrestoadminInstaller,
+                                           YarnSliderModeInstaller],
+        YARN_SLIDER_PRESTO_ADMIN_AND_SLIDER_CLUSTER: [PrestoadminInstaller,
```

Comment:
I don't love the rename. This is adding a little more explicitness at the expense of reasonable length variable names than I'd like.

How about keeping PA abbreviated and leaving the expansion of YARN_SLIDER? Colloquially, presto admin is pa, and if you're writing code here you probably know that. Conversely, yarn and slider apply only to a subset of the people hacking the pa code.